core/storage/config/gae_models_test.py

```diff
@@ -72,6 +72,22 @@ def test_commit(self) -> None:
 
         self.assertEqual(retrieved_model2.value, 'd')
 
+    def test_get_model_association_to_user(self) -> None:
+        self.assertEqual(
+            config_models.ConfigPropertyModel.get_model_association_to_user(),
+            base_models.MODEL_ASSOCIATION_TO_USER.NOT_CORRESPONDING_TO_USER
+        )
+
+    def test_get_export_policy(self) -> None:
+        sample_dict = base_models.VersionedModel.get_export_policy()
+        sample_dict.update({
+            'value': base_models.EXPORT_POLICY.NOT_APPLICABLE
+        })
```

Comment:
I'm worried that this is too similar to the implementation of `get_export_policy()` here:

https://github.com/IamEzio/oppia/blob/14cd0187601d941c6e45ebb33391ce07e005cece/core/storage/base_model/gae_models.py#L1535

For example, if we changed the behavior of `VersionedModel.get_export_policy` but forgot that `ConfigPropertyModel` shouldn't inherit that change, this test wouldn't catch our error.

Maybe it would be better to check the entire export policy dictionary?

Reply:
Done!